foo/lib/config/configloader.py

```diff
@@ -0,0 +1,98 @@
+import os
+from typing import Optional
+
+import boto3
+import yaml
+
+
+class ConfigLoaderException(Exception):
```

Comment:
How about keeping config-specific-exceptions in a separate Exceptions file?
Also how about specifying some specific EXIT_CODE or msg ?